src/Utils.js

```diff
@@ -128,3 +136,46 @@ export function AND(fns) {
     return true;
   };
 }
+
+export const attributePropMap = {
```

Comment:
I did find one. The only potential problem is that react supports some non-standard attributes, which aren't covered in the package I found, and likely wouldn't be covered by other packages.